### DISCUSSION TITLE
How to import a big logbook from another software

### DISCUSSION THREAD

**Discussion by ea5wa** [magicbug/Cloudlog#1937] [Q&A] [answered]
That's my question....

I want to import my log (form another software) but the file is 5MB size.

Is there any tool to split an adif file in different 2MB files?

Thanks

73 de EA5WA Juan Carlos

**Comment by AndreasK79** [comment, marked as answer]
Hi. 5MB ADIF file is not very big. Did you try to import it? I have imported bigger files than that.

**Reply by ea5wa** [reply]
I tried and got a message error "2MB maximum permitted"

**Reply by magicbug** [reply]
This will be due to server configuration which can be changed

**Reply by ea5wa** [reply]
Where? Wich file to edit?

**Reply by rayslinky** [reply]
You need to update upload_max_filesize in php.ini (depending on where your os puts it -- usually under /etc/php).

**Reply by ea5wa** [reply]
Ok. Many thanks... Is a wamp server. Route should be c:/wamp64/bin/php/php8.2.0 but still same message after restarting services

**Reply by ea5wa** [reply]
Finally found the right path. C:/wamp64/bin/apache/apache2.4.54.2/bin/php.ini

Many thanks

**Reply by AlbrechtDIETRICH** [reply]
How to divide the ADIF-File:
 1. Makup your ADIF file.
 2. Open the file and go to 1/3 of the lines, delete all lines after one tof the <EOR> you see.
 3. Save the file e. g. as myADIF_1.adi
 4. Get the 2nd third of the original ADIF-File. 
 5. Store it in a new file e. g. called myADIF_2.adi
 6. Add the header to from the original ADFI-File at he beginning of myADIF_2.adi.
     The header shoulkd look like this and MUST end with <EOH>:
     Cloudlog ADIF export
    <ADIF_VER:5>3.1.4
    <PROGRAMID:8>Cloudlog
    <PROGRAMVERSION:5>2.4.2
    <EOH>
7. Proceed with the 3rd part of the original AFIF file the same way.
8. Upload one ADIF-file after the other to CloudLog.

---> Done.

In case of any questions do not hesitate to contact me.
